### DISCUSSION TITLE
PaddleOCR后续有考虑增加识别图片文字是否加粗的功能吗？

### DISCUSSION THREAD

**Discussion by ZhuangPeiyu** [PaddlePaddle/PaddleOCR#12372] [Ideas & Features]
请问PaddleOCR后续有考虑增加识别图片文字是否加粗的功能吗？

**Comment by tink2123** [comment]
您好，请问是什么场景会需要这个功能呢～～

**Comment by ZhuangPeiyu** [comment]
进行版面还原的任务的时候需要用到这个功能，因为有些文字会被加粗，这个时候如果能识别文字的同事把文字是否加粗的结果也一同返回，后续版面还原的时候效果会更好。

**Comment by TingquanGao** [comment]
目前没有支持该功能的计划，我们后续会评估，谢谢关注~
